topics/valkey-bundle.md

```diff
@@ -0,0 +1,173 @@
+    ```
+ 
+3. **Connect to the server using valkey-cli**
+    
+    To interact with the server, use the Valkey command-line interface (valkey-cli). The example below runs the CLI from a separate container on the same Docker network.
+     
+    ```bash 
+    docker run -it --network some-network --rm valkey/valkey-extensions valkey-cli -h my-valkey-extensions
+    ```
+    
+    This approach is ideal when you want to isolate your CLI session or connect from other containers. However, it does require both containers to be connected to the same Docker network.
+
+    Alternatively, you can run the CLI directly inside the server container: 
+
+    ```bash
+    docker exec -it my-valkey-extensions valkey-cli
+    ```
+
+    This runs the valkey-cli directly inside the server container and connects to the server on localhost. This approach requires no extra setup but only works when the server container is running locally and accessible from your host environment.
+
+4. **Try some commands**
+
+    ```
+    # Check Loaded Modules
+
+    MODULE LIST
+
+    # Use the Valkey JSON Module
+    JSON.SET test $ ‘{"hello": "world"}’
+    JSON.GET test
+
+    # Use the Valkey Bloom Module
+    BF.RESERVE test_bloom 0.01 1000
+    BF.ADD test_bloom "item1"
+    BF.EXISTS test_bloom "item1"
+
+    # Use the Valkey Search Module
+    FT.CREATE index ON HASH PREFIX 1 doc: SCHEMA name TEXT
+    HSET doc:1 name "valkey bundle"
+    FT.SEARCH index "bundle"
+    ```
+
+    Make sure to check out the full list of commands for all the bundle components:
+
+    1. [Valkey Commands](https://valkey.io/commands/)
+    2. [Valkey JSON Commands](https://valkey.io/commands/#json)
+    3. [Valkey Bloom Commands](https://valkey.io/commands/#bloom)
+    4. [Valkey Search Commands](https://valkey.io/commands/#search)
+
+Valkey Bundle supports more advanced setup options too including:
+
+1. **Persistent storage**
+    
+    Persistent storage allows you to save data snapshots locally. The command below is an example of how you can save a snapshot every 60 seconds if at least one write occurred. 
+    
+    ```
+    bash docker run --name my-valkey-extensions -d valkey/valkey-extensions valkey-server --save 60 1 --loglevel warning
+    ```
+    
+2. **Custom Flags with Environment Variable**
+    
+    This allows you to pass additional Valkey flags at runtime using the VALKEY_EXTRA_FLAGS environment variable. This is a flexible way for customizing behavior without needing to modify the existing image or use a custom configuration file. 
+    
+    ```
+    bash docker run --env VALKEY_EXTRA_FLAGS='--save 60 1 --loglevel warning' valkey/valkey-extensions
+    ```
+    
+3. **Use a Custom Configuration File**
+    
+    If you need full control over your Valkey settings, you can create a customer configuration file that will be used inside the container. This allows you to override all default settings through the use of your own valkey.conf file.
```

Comment:
This step is done without context. You are assuming there is a directory (/myvalkey/conf) that you are mounting in the container. This folder must have a valkey.conf file. You should explain what's happening here.